docs/build.rb

```diff
@@ -200,11 +200,15 @@ def extract_absolute_wiki_links(markdown)
       new_links
     end
 
-    # Scans for substrings such as '[[Reference Sites|Git Reference Sites]]'
+    # Scans for Github wiki flavor links such as:
+    #   '[[Relative Path]]'
+    #   '[[Custom name|Relative Path]]'
+    #   '[[Custom name|relative-path]]'
+    #   '[[Custom name|./relative-path.md]]'
     def extract_relative_links(markdown)
       existing_links = @links
       new_links = {}
-      markdown.scan(/(\[\[([\w_ '().:,-]+)(?:\|([\w_ '():,.-]+))?\]\])/) do |full_match, left, right|
+      markdown.scan(/(\[\[([\w\/_ '().:,-]+)(?:\|([\w\/_ '():,.-]+))?\]\])/) do |full_match, left, right|
```

Comment:
Required to catch `/` in relative paths such as `[[Custom name|./relative-path.md]]`